frontend/src/components/Thread.vue

```diff
@@ -53,16 +69,34 @@ export default {
     },
   },
   methods: {
+    // TODO: this is currently not used, because I can't figure out a good place to put this
```

Comment:
okay to leave it, can you specify what `this` is referring to just to make sure (assuming it's resolving threads)

Reply:
Yup, will specify.